packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -25,8 +25,10 @@ const { keys, values, fromEntries } = Object;
  * @returns {Promise<Record<string, V>>}
  * @template V
  */
-const allValues = async obj =>
-  fromEntries(zip(keys(obj), await Promise.all(values(obj))));
+const allValues = async obj => {
```

Comment:
Much better, thanks!

Reply:
It looks like your force pushes since then have erased https://github.com/Agoric/agoric-sdk/commit/667ad5c9cafc64e922a637702751f69b259d0f9f

I've pushed it again. Mind pulling f902c5d27 your local branch?